src/item_pocket.cpp

```diff
@@ -835,8 +835,8 @@ void item_pocket::general_info( std::vector<iteminfo> &info, int pocket_number,
 
     // Show volume/weight for normal containers, or ammo capacity if ammo_restriction is defined
     if( data->ammo_restriction.empty() ) {
-        info.push_back( vol_to_info( "CONTAINER", _( "Volume: " ), volume_capacity() ) );
-        info.push_back( weight_to_info( "CONTAINER", _( "  Weight: " ), weight_capacity() ) );
+        info.push_back( vol_to_info( "CONTAINER", _( "Volume: " ), volume_capacity(), 2, false ) );
```

Comment:
```suggestion
        iteminfo volume_info = vol_to_info("CONTAINER", _("Volume: "), volume_capacity());
        volume_info.bLowerIsBetter = true;
        info.push_back( volume_info ); );
```

Reply:
If you flip the bool here before sending if off into the `std::vector<iteminfo> info` then you can make sure it has the intended behavior on pockets without it affecting the behavior on other volumes.

Reply:
not sure i understand ... you want to color volume for amunition containers red when they are bigger?
Isn't that the wrong way around?

Reply:
Oh you're right that's weird. I got turned around trying to keep track of flags and bools. 

I suppose my suggested change would instead be for all the cases that aren't pockets, magazines, or containers.